jest.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
   moduleFileExtensions: ["ts", "js"],
-  collectCoverageFrom: ["src/*.ts"],
+  collectCoverageFrom: ["src/*.ts", "!src/node-web-compat-web.ts"],
```

Comment:
Is the exclusion because Cypress tests are covering the unit tests for the code here?

Reply:
Yes indeed